maxim/logger/components/utils.py

```
@@ -0,0 +1,84 @@
+"""
+This module contains utility functions for parsing attachments from messages.
+"""
+
+import base64
+import re
+from typing import TYPE_CHECKING, Tuple
```

Ruff (0.11.9)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)